io/nvmf/nvmftest.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2018 IBM
+# Author: Narasimhan V <sim@linux.vnet.ibm.com>
+
+"""
+NVM Express over Fabrics defines a common architecture that supports a
+range of storage networking fabrics for NVMe block storage protocol over
```

Comment:
`storage network fabrics` look me more appropriate

Reply:
@PraveenPenguin This is from: http://www.nvmexpress.org/wp-content/uploads/NVMe_Over_Fabrics.pdf

Reply:
ah sure i just go with how we pronounce

Reply:
Agreed, but since this is how it is in the community, I will stick with this :-)